<ui version="4.0" >
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>461</width>
    <height>397</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string/>
  </property>
  <property name="sizeGripEnabled" >
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout" >
   <item row="0" column="0" >
    <widget class="QTabWidget" name="tabWidget" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Preferred" hsizetype="Preferred" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="currentIndex" >
      <number>0</number>
     </property>
     <widget class="QWidget" name="tabUniform" >
      <attribute name="title" >
       <string>Uniforms</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2" >
       <item row="0" column="0" >
        <widget class="QComboBox" name="comboUniforms" />
       </item>
       <item row="1" column="0" >
        <widget class="QGroupBox" name="BoxUnifProps" >
         <property name="title" >
          <string/>
         </property>
         <property name="checkable" >
          <bool>false</bool>
         </property>
         <layout class="QGridLayout" name="gridLayout_6" />
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabTextures" >
      <attribute name="title" >
       <string>Textures</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_5" >
       <item row="0" column="0" >
        <widget class="QComboBox" name="comboTextures" />
       </item>
       <item row="1" column="0" >
        <widget class="QGroupBox" name="BoxTextureProps" >
         <property name="title" >
          <string/>
         </property>
         <layout class="QGridLayout" name="gridLayout_8" >
          <item row="0" column="0" >
           <layout class="QGridLayout" name="gridLayout_7" >
            <item row="0" column="0" >
             <layout class="QHBoxLayout" name="horizontalLayout" >
              <item>
               <widget class="QLineEdit" name="EditTexFile" >
                <property name="readOnly" >
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="btnChangeTexture" >
                <property name="text" >
                 <string>Browse</string>
                </property>
                <property name="autoDefault" >
                 <bool>false</bool>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item row="1" column="0" >
             <layout class="QHBoxLayout" name="horizontalLayout_2" >
              <item>
               <widget class="QGroupBox" name="TexInfoBox" >
                <property name="title" >
                 <string/>
                </property>
                <layout class="QGridLayout" name="gridLayout_9" >
                 <item row="0" column="0" >
                  <layout class="QHBoxLayout" name="horizontalLayout_3" >
                   <item>
                    <widget class="QLabel" name="lblPreview" >
                     <property name="maximumSize" >
                      <size>
                       <width>100</width>
                       <height>100</height>
                      </size>
                     </property>
                     <property name="frameShape" >
                      <enum>QFrame::NoFrame</enum>
                     </property>
                     <property name="frameShadow" >
                      <enum>QFrame::Plain</enum>
                     </property>
                     <property name="text" >
                      <string/>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <layout class="QVBoxLayout" name="infoTexLayout" />
                   </item>
                  </layout>
                 </item>
                </layout>
               </widget>
              </item>
             </layout>
            </item>
            <item row="2" column="0" >
             <widget class="QTableWidget" name="TexStatesTable" >
              <property name="autoScroll" >
               <bool>false</bool>
              </property>
              <property name="wordWrap" >
               <bool>false</bool>
              </property>
              <property name="columnCount" >
               <number>0</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabStates" >
      <attribute name="title" >
       <string>GL States</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_10" >
       <item row="0" column="0" >
        <widget class="QTableWidget" name="glStatesTable" >
         <property name="autoScroll" >
          <bool>false</bool>
         </property>
         <property name="wordWrap" >
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabVertex" >
      <attribute name="title" >
       <string>Vertex Program</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_3" >
       <item row="0" column="0" >
        <widget class="QTextEdit" name="textVert" >
         <property name="sizePolicy" >
          <sizepolicy vsizetype="Preferred" hsizetype="Preferred" >
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="lineWrapMode" >
          <enum>QTextEdit::NoWrap</enum>
         </property>
         <property name="readOnly" >
          <bool>true</bool>
         </property>
         <property name="textInteractionFlags" >
          <set>Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabFragment" >
      <attribute name="title" >
       <string>Fragment Program</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_4" >
       <item row="0" column="0" >
        <widget class="QTextEdit" name="textFrag" >
         <property name="sizePolicy" >
          <sizepolicy vsizetype="Preferred" hsizetype="Preferred" >
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="lineWrapMode" >
          <enum>QTextEdit::NoWrap</enum>
         </property>
         <property name="readOnly" >
          <bool>true</bool>
         </property>
         <property name="textInteractionFlags" >
          <set>Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
